arch/arm/boot/dts/aspeed/aspeed-g6.dtsi

```diff
@@ -1146,3 +1159,163 @@
+                pinctrl-0 = <&pinctrl_i3c2_default>;
+
+                i2c-scl-hz = <1000000>;
+                i3c-scl-hz = <2000000>;
+                i3c-od-scl-hi-period-ns = <380>;
+                i3c-od-scl-lo-period-ns = <620>;
+                i3c-pp-scl-hi-period-ns = <250>;
+                i3c-pp-scl-lo-period-ns = <250>;
+                sda-tx-hold-ns = <70>;
+                aspeed,global-regs = <&i3c_global 1>;
+                sda-pullup-ohms = <2000>;
+
+                status = "disabled";
+        };
+
+        i3c2: i3c@4000 {
+                #address-cells = <3>;
+                #size-cells = <0>;
+                #interrupt-cells = <1>;
+
+                reg = <0x4000 0x1000>;
+                compatible = "aspeed,ast2600-i3c";
+                clocks = <&syscon ASPEED_CLK_GATE_I3C2CLK>;
+                resets = <&syscon ASPEED_RESET_I3C2>;
+                interrupts = <GIC_SPI 104 IRQ_TYPE_LEVEL_HIGH>;
+
+                i2c-scl-hz = <1000000>;
+                i3c-scl-hz = <2000000>;
+                i3c-od-scl-hi-period-ns = <380>;
+                i3c-od-scl-lo-period-ns = <620>;
+                i3c-pp-scl-hi-period-ns = <250>;
+                i3c-pp-scl-lo-period-ns = <250>;
+                sda-tx-hold-ns = <70>;
+                aspeed,global-regs = <&i3c_global 2>;
+                sda-pullup-ohms = <2000>;
+
+                status = "disabled";
+        };
+
+        i3c3: i3c@5000 {
+                #address-cells = <3>;
+                #size-cells = <0>;
+                #interrupt-cells = <1>;
+
+                reg = <0x5000 0x1000>;
+                compatible = "aspeed,ast2600-i3c";
+                clocks = <&syscon ASPEED_CLK_GATE_I3C3CLK>;
+                resets = <&syscon ASPEED_RESET_I3C3>;
+                interrupts = <GIC_SPI 105 IRQ_TYPE_LEVEL_HIGH>;
+
+                i2c-scl-hz = <1000000>;
+                i3c-scl-hz = <2000000>;
+                i3c-od-scl-hi-period-ns = <380>;
+                i3c-od-scl-lo-period-ns = <620>;
+                i3c-pp-scl-hi-period-ns = <250>;
+                i3c-pp-scl-lo-period-ns = <250>;
+                sda-tx-hold-ns = <70>;
+                aspeed,global-regs = <&i3c_global 3>;
+                sda-pullup-ohms = <2000>;
+
+                status = "disabled";
+        };
+
+        i3c4: i3c@6000 {
+                #address-cells = <3>;
+                #size-cells = <0>;
+                #interrupt-cells = <1>;
+
+                reg = <0x6000 0x1000>;
+                compatible = "aspeed,ast2600-i3c";
+                clocks = <&syscon ASPEED_CLK_GATE_I3C4CLK>;
+                resets = <&syscon ASPEED_RESET_I3C4>;
+                interrupts = <GIC_SPI 106 IRQ_TYPE_LEVEL_HIGH>;
+                pinctrl-names = "default";
+                pinctrl-0 = <&pinctrl_i3c5_default>;
+
+                i2c-scl-hz = <1000000>;
+                i3c-scl-hz = <2000000>;
+                i3c-od-scl-hi-period-ns = <380>;
```

Comment:
How did we arrive at these values for hi and lo?

Reply:
This is copy of aspeed version.

Reply:
Ok, in driver we may need to override tx hold and timing delay later.

#define SDA_DLY_TIMING_OFFSET        0xD0
#define SDA_TX_HOLD_APML             0x00030000
#define SDA_TX_HOLD_DIMM             0x00070000